src/lib/cloudfuse/child_process_windows.cpp

```diff
@@ -77,6 +77,20 @@ allow-other: true
   region: { AWS_REGION }
 )";
 
+// Return available drive letter to mount
+std::string getAvailableDriveLetter()
+{
+    DWORD driveMask = GetLogicalDrives();
+    for (char letter = 'Z'; letter >= 'A'; --letter)
+    {
+        if (!(driveMask & (1 << (letter - 'A'))))
```

Comment:
can you explain what is happening in this conditional?

Reply:
Yeah so GetLogicalDrives returns a bitmask of which drives are available. So bit position 0 is the A drive, bit position 1 is drive B, and so on. Then if the if check we use a bitwise and to see if that letters position in the bitmask was set. If not then the drive is available so we can use that drive letter.